### DISCUSSION TITLE
please simCLR

### DISCUSSION THREAD

**Discussion by 1234532314342** [open-mmlab/mmclassification#1470] [Q&A]
I want to ask, isn't pre-training simCLR unsupervised? Why the dataset also needs input labels

**Comment by fangyixiao18** [comment]
> I want to ask, isn't pre-training simCLR unsupervised? Why the dataset also needs input labels

It doesn't need input labels, even if the dataset loaded the labels, the algorithms would not use it.

**Reply by 1234532314342** [reply]
Thank you very much ！If my backbone is modified from resnet50, will my resnet50 need to be modified when I pre-train?
